components/merkledb/src/db.rs

```diff
@@ -565,23 +564,15 @@ impl Snapshot for FlushedFork {
 }
 
 impl Fork {
-    /// Finalizes all changes after the latest checkpoint.
-    ///
-    /// # Panics
-    ///
-    /// Panics if there is no active checkpoint, or the latest checkpoint
-    /// is already committed or rolled back.
+    /// Finalizes all changes that was made after previous execution of itself.
+    /// If no `flush` method was called before, finalizes all changes that was
+    /// made after creation of `Fork`. In other words creates checkpoint.
```

Comment:
/// Finalizes all changes that were made after previous execution of the `flush` method.
 /// If no `flush` method had been called before, finalizes all changes that were
/// made after creation of `Fork`.